op-chain-ops/scripts/assert_withdrawals_exist

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
```

Comment:
```suggestion
#!/bin/bash
set -uo pipefail

```

Probably don't want `-e` here since we're explicitly testing the return code.  The downside of the current approach though is that if `cast` fails because the node is offline the script will just print `Error: Withdrawals are null` which is potentially misleading.  Probably not a big deal if cast prints an error to stderr as well though.

Reply:
yea, I did test with the whole set -euo pipefail and did notice it didn't do what I wanted. Will add this and see if it works

Reply:
done. Works. Also set the proper exit code.